src/components/remotecontrol/remotecontrol.js

```diff
@@ -207,7 +207,18 @@ define(['browser', 'datetime', 'backdrop', 'libraryBrowser', 'listView', 'imageL
             });
             setImageUrl(context, state, url);
             if (item) {
-                backdrop.setBackdrops([item]);
+                if (item.Type === 'Audio') {
+                    var imgUrl = apiClient.getScaledImageUrl(item.Id, {
+                        type: 'Primary',
+                        maxWidth: dom.getScreenWidth(),
+                        index: 0,
+                        tag: item.AlbumPrimaryImageTag[0]
```

Comment:
This doesn't work for me. No image: 404 error.
It generates URL for image of `item.Id`, but should be `item.AlbumId`, if one exists.
Looks like `AlbumPrimaryImageTag` might just be a string. And in this case, `index` shouldn't be specified.

Reply:
> folder.jpg

If @dkanada can confirm, I don't think we get these right now? Or there were some weird stuff with them?

Reply:
> > folder.jpg
> 
> If @dkanada can confirm, I don't think we get these right now? Or there were some weird stuff with them?

I have them already in metadata.
Probably this https://github.com/jellyfin/jellyfin/blob/e3db0ac400e7a1a42e7a844a13ab052d26125612/MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs#L29-L35

Reply:
LocalImageProvider is somehow different from the method a lot of the images use, it's quite annoying. Not sure how that would affect the web client though.